components/Footer/index.tsx

```diff
@@ -1,68 +1,26 @@
 import React from "react";
-import { Box, Link, Theme, Typography, useTheme } from "@mui/material";
+import classes from "./style.module.scss";
 
-const Footer: React.FC = () => {
-    const theme = useTheme<Theme>();
-
-    return (
-        <Box
-            component={"footer"}
-            display={"flex"}
-            flexDirection={"column"}
-            alignItems={"flex-start"}
-            flexWrap={"nowrap"}
-            sx={{
-                "& *": {
-                    marginTop: theme.spacing(1),
-                    marginBottom: theme.spacing(1),
-                },
-            }}
-        >
-            <Typography
-                variant={"body2"}
-                component={"span"}
-                color={"#999"}
-                fontSize={".6rem"}
-                marginRight={4}
-                whiteSpace={"nowrap"}
-                overflow={"hidden"}
-                textOverflow={"ellipsis"}
-                maxWidth={"100%"}
-            >
-                {"Copyright © 2021 "}
-                <Link
-                    href="/"
-                    rel="self"
-                    color={"inherit"}
-                    underline={"hover"}
-                    target={"_self"}
-                >
-                    {"Hainan Ungine Times Technology Co.,Ltd."}
-                </Link>
-                {" All Right Reserved."}
-            </Typography>
-            <Typography
-                variant={"body2"}
-                component={"span"}
-                color={"#999"}
-                fontSize={".6rem"}
-                whiteSpace={"nowrap"}
-                overflow={"hidden"}
-                textOverflow={"ellipsis"}
-                maxWidth={"100%"}
+const Footer: React.FC = () => (
+    <footer className={classes.root}>
+        <p className={classes.copyright}>
+            {"Copyright © 2021 "}
```

Comment:
Maybe we should add '-{thisYear}'

Reply:
https://github.com/Ungine-Tech/Homepage-Ungine/blob/a0b1ef57b0826f8dd63ad8da3ff1306051aff17c/components/Footer/index.tsx#L6-L12